src/main.py

```diff
@@ -79,23 +80,16 @@ def render_page(sensors):
     return html
 
 
-# setup a socket to listen on port 80
-s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-s.bind(('', 80))
-s.listen(5)
 
-print("Bound to port 80")
 
 sensors = [Blinker("Blinker", 2), Barometer("Barometer", 5, 4),
            AirQuality("Air Quality", 0)]
 
-while True:
-    # wait on a new connection
-    conn, addr = s.accept()
-    # f-strings allow to include formatted vars into strings
-    print(f"Got a connection from {addr}")
+
+async def serve(reader, writer):
```

Comment:
The callback for `uasyncio.start_server`. Should have two parameters, one for reader, one for write, both of type `Stream`. From and to these we can read async and write async.